.build/run-python-dtests.sh

```diff
@@ -74,7 +74,7 @@ if [ "${java_version}" -eq 17 ] && [[ "${target}" == "dtest-upgrade" ]] ; then
 fi
 
 python_version=$(python -V 2>&1 | awk '{print $2}' | awk -F'.' '{print $1"."$2}')
-python_regx_supported_versions="^(3.8|3.9|3.10|3.11)$"
+python_regx_supported_versions="^(3.6|3.7|3.8|3.9|3.10|3.11)$"
```

Comment:
the docker images also need to be updated for this to work. 
https://github.com/apache/cassandra/blob/cassandra-5.0/.build/docker/ubuntu2004_test.docker

Reply:
and…  this script has nothing to do with the cqlsh tests :)

Reply:
I think this bit just needs to be reverted.

Reply:
Done